src/main/java/com/coveo/pushapiclient/PushSource.java

```diff
@@ -125,13 +125,53 @@ public static PushSource fromPlatformUrl(
     return new PushSource(apiKey, organizationId, sourceId, platformUrl);
   }
 
+  /**
+   * Create a Push source instance
+   *
+   * @param apiKey The API key used for all operations regarding your source.
+   *     <p>Ensure your API key has the required privileges for the operation you will be performing
+   *     *
+   *     <p>For more information about which privileges are required, see <a href=
+   *     "https://docs.coveo.com/en/1707#sources-domain">Privilege Reference.</a>
+   * @param organizationId The unique identifier of your organization.
+   *     <p>The Organization Id can be retrieved in the URL of your Coveo organization.
+   * @param sourceId The unique identifier of the target Push source.
+   *     <p>The Source Id can be retrieved when you edit your source in the <a href=
+   *     "https://docs.coveo.com/en/183/glossary/coveo-administration-console">Coveo Administration
+   *     Console</a>
+   * @param platformUrl The object containing additional information on the URL endpoint. You can
+   *     use the {@link PlatformUrl} when your organization is located in a non-default Coveo
+   *     environement and/or region. When not specified, the default platform URL values will be
+   *     used: {@link PlatformUrl#DEFAULT_ENVIRONMENT} and {@link PlatformUrl#DEFAULT_REGION}
+   *     * @param options The configuration options for exponential backoff
+   */
+  public static PushSource fromPlatformUrl(
```

Comment:
You forgot the doc for Backoff options

Reply:
Whoops! I didn't forget, I just didn't format it properly
Line 146 🤦 
```
*     * @param options The configuration options for exponential backoff
```